.github/workflows/lint_legacy_python.yml

```
@@ -0,0 +1,19 @@
+name: lint_legacy_python
+on: [pull_request, push]
+jobs:
+   lint_legacy_python:
+     runs-on: ubuntu-latest
+     steps:
+       - uses: actions/checkout@v2
```

```suggestion
       - uses: actions/checkout@v3
```
Has just been released 🙂: https://github.com/actions/checkout/releases